src/solver/conv_asm_implicit_gemm_gtc_perf_config.cpp

```diff
@@ -213,16 +213,19 @@ void PerformanceConfigAsmImplicitGemmGTC::CopyParameters(
 std::string PerformanceConfigAsmImplicitGemmGTC::ToString() const
 {
     std::ostringstream ss;
-    ss << ToKernelName();
+    // ss << ToKernelName();
+    Serialize(ss);
     if(gemm_k_global_split != 0)
         ss << "[" << gemm_k_global_split << "]";
     return ss.str();
 }
-std::string PerformanceConfigAsmImplicitGemmGTC::ToKernelName() const
+std::string PerformanceConfigAsmImplicitGemmGTC::ToKernelName(const ConvolutionContext& ctx) const
 {
     std::ostringstream kernel_name;
     std::string kernel_precision = precision == miopenFloat ? "fp32" : "fp16";
-    kernel_name << "igemm_" << direction << "_gtcx_" << tensor_layout << "_" << kernel_precision
+    const auto device_name = ctx.GetStream().GetDeviceName();
+    std::string kernel_arch = device_name == "gfx908" ? "_gtcx_" : "_gtcx2_";
+    kernel_name << "igemm_" << direction << kernel_arch << tensor_layout << "_" << kernel_precision
```

Comment:
`kernel_arch` can change to like `gtc`? since here `_gtcx_` or `_gtcx2_` is not exactly arch name

Reply:
yes. changed.